modules/appeals_api/app/services/appeals_api/pdf_construction/supplemental_claim/v2/form_fields.rb

```diff
@@ -45,85 +45,94 @@ def insurance_policy_number
+          def claimant_type
+            'form1[0].#subform[2].RadioButtonList[1]'
           end
 
-          def date_of_record
-            'form1[0].#subform[3].DATEOFTREATMENTRECORDS1[0]'
+          def benefit_type
+            'form1[0].#subform[2].RadioButtonList[0]'
           end
 
-          def notice_of_acknowledgement_no
-            'form1[0].#subform[3].TIME2TO430PM[0]'
-          end
-
-          def notice_of_acknowledgement_yes
-            'form1[0].#subform[3].TIME1230TO2PM[0]'
+          def soc_ssoc_opt_in
+            'form1[0].#subform[2].RadioButtonList[2]'
           end
 
           def date_signed
             'form1[0].#subform[3].DATESIGNED[0]'
           end
 
-          def date_signed_alternate_signer
-            'form1[0].#subform[3].DATESIGNED[1]'
-          end
-
+          # rubocop:disable Metrics/MethodLength
           def boxes
-            # number_of_issue_boxes = 7
-            # number_of_new_evidence_submission_boxes = 3
+            number_of_issues_on_form = Structure::MAX_NUMBER_OF_ISSUES_ON_MAIN_FORM
+            number_of_evidence_submission_boxes = Structure::MAX_NUMBER_OF_EVIDENCE_LOCATIONS_FORM
+
             {
               # PAGE 3 '#subform[2]'
-              veteran_first_name: { at: [3, 589], width: 195 },
-              veteran_last_name: { at: [238, 589], width: 300 },
-              address_number_street: { at: [23, 420], width: 510 },
-              address_apartment_or_unit_number: { at: [60, 399], width: 78 },
-              address_city: { at: [194, 399], width: 305 },
-              email: { at: [286, 343], width: 244 },
-              contestable_issue: { at: [0, 207], width: 402, height: 22 }, # first text box only
-              soc_date: { at: [435, 208], width: 80 }, # first text box only
-
-              # PAGE 4 '#subform[3]'
-              new_evidence_name_location: { at: [0, 207], width: 404, height: 36 }, # first text box only
-              signature_of_veteran_claimant_or_rep: { at: [0, 260], width: 408 },
-              print_name_veteran_claimaint_or_rep: { at: [0, 235], width: 540 },
-              signature_alternate_signer: { at: [0, 98], width: 408 },
-              print_name_alternate_signer: { at: [0, 73], width: 540 }
+              veteran_first_name: { at: [3, 592], width: 195 },
+              veteran_last_name: { at: [238, 591], width: 300 },
+              mailing_address_number_and_street: { at: [23, 423], width: 510 },
+              mailing_address_apartment_or_unit_number: { at: [60, 401], width: 78 },
+              mailing_address_city_and_box: { at: [195, 402], width: 308 },
+              email: { at: [286, 347], width: 244 },
+
+              # doesn't yet print extras on an additional page
+              contestable_issues: [].tap do |n|
+                                    number_of_issues_on_form.times do |i|
+                                      n << { at: [0, 203 - (30 * i)], width: 402, height: 22, valign: :top }
```

Comment:
I feel as though these being extracted for readability's sake makes sense. Thoughts?

Reply:
I'll wire it up and see how it looks.